generator.py

```diff
@@ -11,32 +10,29 @@
 NEED_UPDATE = 1
 FAILURE = 2
 
-DEFAULT_GIR_DIRECTORY = 'gir-files'
-DEFAULT_GIR_PATH = './gir/target/release/gir'
+DEFAULT_GIR_FILES = Path('./gir-files')
```

Comment:
I suggest `DEFAULT_GIR_FILES_DIRECTORY` instead to be consistent with `DEFAULT_GIR_DIRECTORY`.

Reply:
Makes sense, will update :)

Reply:
How about the command-line option, is `--gir-files` okay with you?

Reply:
The option yes, but the field not really. Could you change it to `gir_files_path` please (same as `gir_path` :) )?

Reply:
That name is derived from the flag, butt can be overridden with `dest=`.

On that note I don't find `--gir-path` very expressive, should we perhaps rename that to just `--gir` or `--gir-exe(cutable)`? We can also add some short-hands like `-g`, `-f`, '-y` etc.

Reply:
Please don't add short-hands. I voluntarily didn't add them. As for the name, I find `--gir-path` good enough but others opinions can't hurt. cc @bilelmoussaoui @sdroege

Reply:
--gir-path is good for me as well, but no hard opinions
@sophie-h any opinions?

Reply:
Renamed the field to `gir_files_path`. I'm not using `--gir-path` yet so I have no stake in the choice :)

Reply:
Let's add short versions of the parameters in a separate PR as this apparently is contentious and needs to be discussed, while everything else in this PR is easy. (FWIW I don't care either way)

Reply:
@sdroege Fyi this is not about shorthands as much as it is about the naming of `--gir-path`. But whatever, this is a small development utility, not a public facing API :)

Reply:
I also don't care either way for that one :)